python/py-pyqtgraph/Portfile

```diff
@@ -22,13 +22,15 @@ long_description    PyQtGraph is a pure-python graphics and GUI library \
                     for fast display.
 
 homepage            http://pyqtgraph.org/
-master_sites        http://pyqtgraph.org/downloads
+master_sites        http://pyqtgraph.org/downloads/${version}/
 distname            ${real_name}-${version}
 
-checksums           rmd160  45d5f581afd5ee68f25bf1fabb8f9b89dd97516b \
-                    sha256  4c0589774e3c8b0c374931397cf6356b9cc99a790215d1917bb7f015c6f0729a
+checksums           rmd160  f19baf750e676fabb647f28729cabd9afc48bfb6 \
+                    sha256  c74597dd87c31987099bd0949a5207e5d58297bd8a663279562ca1cb334074fc \
+                    size    1544516
 
-python.versions     27 34 35 36
+
+python.versions     27 36
```

Comment:
Are there any 3.4 or 3.5 packages that depend on this?

Reply:
I do not know. Is there a command-line way to check?

Reply:
I believe this is it:

```port echo depends:py34-pyqtgraph```

It looks like nothing for both py34 and py35 - or py36 for that matter.

Reply:
Shouldn't removed python subports go into py-graveyard?

Reply:
Yes, they should.

Reply:
@jjstickel Please submit a PR adding the python subports to py-graveyard.